src/gestures/drag/VisualElementDragControls.ts

```diff
@@ -357,14 +378,18 @@ export class VisualElementDragControls {
         this.prepareBoundingBox()
         this.cursorProgress.x = 0.5
         this.cursorProgress.y = 0.5
-        this.updateAxis("x", event)
-        this.updateAxis("y", event)
+        this.updateAxis("x", event, 0)
+        this.updateAxis("y", event, 0)
```

Comment:
I haven't fixed this yet, looking at the diff from #623 it looks like I should be able to take advantage of `this.visualElement.boundingBox` right?